src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Lottery();
```

Comment:
It isn't a ball. it's lottery
```suggestion
        Lottery lottery = new Lottery();
```